src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 server.port: 5050
 spring.datasource.driverClassName = org.postgresql.Driver
 spring.datasource.url=jdbc:postgresql://localhost/polls
-spring.datasource.username=agyen
+spring.datasource.username=postgres
```

Comment:
get rid of usernames & passwords
this should be an env variable
consider changing the password (and don't check it in!)

Reply:
username and password now in env var